Makefile

```diff
@@ -410,3 +410,7 @@ vendor/check: vendor/fix
 vendor/fix:
 	go mod tidy
 	go mod vendor
+
+.PHONY: manifest
+manifest:
```

Comment:
Is there a better name that we could use for this make target. `manifest` is very generic and we use the term manifest when talking about the CSV also. If this is for security checks maybe `security/manifest` maybe a better target name.

Reply:
Use of `/` makes me feel like there should be more than one item available but I agree that the `make manifest` isn't best. Maybe `make manifest/release` as later on, there will also be a manifest for current master? WDYT?